### ISSUE TITLE
Make the filled-in love and favorite icons easier to see

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6627]
### Why this would be helpful

The contrast of the outlined heart and star icons are pretty good, but the ones filled in with color have been unchanged and remain inaccessible (especially the star).

### My suggestion

Add an option to `colorblind` that increases the contrast of the heart and star icons.

### Additional context

- Related conversation:
  https://github.com/ScratchAddons/ScratchAddons/issues/5966#issuecomment-1693805489
- Initially part of:
  #6400

**Comment by DNin01** [comment]
Kind of waiting on this:
> @WorldLanguages
> > @DNin01
> > Should this be an expansion to the Customizable link style addon or should it be its own addon?
> 
> Hmm, it might need to be its own addon, anything else wouldn't fit. In the future we can provide a group/category/filter to list all accessibility-related addons, but as you said, they are great customization options for everyone.
> 
> > @DNin01
> > And how would be handle the addon ID in the case that that feature or Customizable link style gets a different addon? Would the existing `colorblind` addon move into a new one or...?
> 
> That addon ID is really unfortunate now, so we may change the ID and add a script that migrates settings. Maybe you should open a different issue about it.

Source: https://github.com/ScratchAddons/ScratchAddons/issues/5966#issuecomment-1694782577

**Comment by rgantzos** [comment]
If there's no outline then that distinguishes between loved/not loved and favorited/not favorited, doesn't it?

**Comment by DNin01** [comment]
> If there's no outline then that distinguishes between loved/not loved and favorited/not favorited, doesn't it?

Yes, but I'm talking about the visibility of the icons, not contrast between states.

**Comment by Secret-chest** [comment]
Make them customisable?

**Comment by WorldLanguages** [comment]
> Kind of waiting on this:

Waiting on what exactly? This would be a new addon, right?

**Comment by rgantzos** [comment]
> > Kind of waiting on this:
> 
> Waiting on what exactly? This would be a new addon, right?

Not according to the tags.

**Comment by DNin01** [comment]
> > > Kind of waiting on this:
> 
> > 
> 
> > Waiting on what exactly? This would be a new addon, right?
> 
> 
> 
> Not according to the tags.

It's up in the air at the moment.

**Comment by WorldLanguages** [comment]
I say you go ahead with the PR? We can change the addon ID before merging the PR.
